ddsenabler_yaml/src/cpp/EnablerConfiguration.cpp

```diff
@@ -43,16 +47,82 @@ using namespace eprosima::ddspipe::participants;
 using namespace eprosima::ddspipe::participants::types;
 using namespace eprosima::ddspipe::yaml;
 
-EnablerConfiguration::EnablerConfiguration(
-        const Yaml& yml)
+// Helper method to recognize if the configuration file is in JSON format
+bool is_json(const std::string& file_path)
```

Comment:
Using the file extension is not very robust I believe, can't se use a method from json or yaml libraries?